lib/sanitize.js

```diff
@@ -0,0 +1,48 @@
+var dompurify = require('dompurify');
+
+
+function addTabindexToIdPolicy(node) {
```

Comment:
We should drop adding `tabIndex` as the `sanitizer` does not do that.

Reply:
removed